pig/src/org/partiql/pig/generator/kotlin/KTypeDomainConverter.kt

```diff
@@ -62,7 +61,7 @@ private class KTypeDomainConverter(
 
         typeDomain.types.forEach {
             when(it) {
-                DataType.Int, DataType.Symbol, DataType.Ion -> { /* intentionally blank */ }
+                DataType.Int, DataType.Symbol, DataType.Ion, DataType.Bool -> { /* intentionally blank */ }
```

Comment:
Also a minor addition that stopped my IDE from complaining about missing a case.